frontend/app/src/components/home/MemeFighterContent.svelte

```diff
@@ -0,0 +1,226 @@
+<svelte:options immutable />
```

Comment:
A lot of code common to ImageContent - could this somehow be re-used by having an image withg an empty caption?

Reply:
Yes - I wasn't sure how similar they would be but you're right, they are all but the same.